README.md

```diff
@@ -1,70 +1,206 @@
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
 
-### `npm test`
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+# 📖 REACT METRICWEBAPP  <a name="about-project"></a>
 
-### `npm run build`
+**REACT METRICWEBAPP** is a webApp that contains Apple product annual revenue, displaying them in segment of year, and into a smaller segment of each country revenue. Here is a [link](https://www.loom.com/share/16a20c8d07414289a6e4e871de6f7242) to a video about the project.
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+## 🛠 Built With <a name="built-with"></a>
+- REACT
+- CSS
+- REDUX
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+### Tech Stack <a name="tech-stack"></a>
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+ <details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https:/reactjs.com/">REACTJS</a></li>
+  </ul>
+</details> 
+<details>
+  <summary>style</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">CSS</a></li>
+  </ul>
+</details>
 
-### `npm run eject`
+<details>
+<summary>state management</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">REDUXJS</a></li>
+  </ul>
+</details>
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+### Key Features <a name="key-features"></a>
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+- **receive data from an API using await fetch**
+- **manage state using redux**
+- **create a Home page to display the yearly data**
+- **Create a page for each data using useparams.**
 
-## Learn More
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
 
-To learn React, check out the [React documentation](https://reactjs.org/).
 
-### Code Splitting
+## 🚀 Live Demo <a name="live-demo"></a>
```

Comment:
- [optional] Great job with the deployed link.  you were also required to provide a project presentation video with the following 
 a video presenting the features of the project.
The maximum length of the video is 5 minutes.
The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
The video contains a demo of the project, highlighting all the project features.
    ---  all this is to help you learn more and gain extra skills

Reply:
The video demo is already presented below in the README file and I viewed and tested it.